src/pages/editor/EditorInstitution.tsx

```diff
@@ -61,97 +62,149 @@ export const EditorInstitution = () => {
-
-          <Typography variant="h3" component="h2">
-            {t('basic_data.institutions.ror')}
-          </Typography>
-          <Typography paragraph>
-            {customer?.rorId ? (
-              <Link href={customer.rorId} target="_blank" rel="noopener noreferrer">
-                {customer.rorId}
+        <Grid container spacing={2}>
+          <Grid container item xs={12} spacing={2}>
+            <Grid item xs={12} md={4}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.institution_name_norwegian')}
+              </Typography>
+              <Typography>{institution?.labels.nb ?? '-'}</Typography>
+            </Grid>
+
+            <Grid item xs={12} md={4}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.institution_name_english')}
+              </Typography>
+              <Typography>{institution?.labels.en ?? '-'}</Typography>
+            </Grid>
+
+            <Grid container item xs={8} md={4}>
+              <Grid item xs={6}>
+                <Typography variant="h3" component="h2">
+                  {t('editor.institution.institution_short_name')}
+                </Typography>
+                <Typography>{organizationQuery.data?.acronym ?? '-'}</Typography>
+              </Grid>
+
+              <Grid item xs={6}>
+                <Typography variant="h3" component="h2">
+                  {t('editor.institution.institution_code')}
+                </Typography>
+                <Typography>{institution?.id.split('/').pop() ?? '-'}</Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+
+          <Grid container item xs={12} spacing={2}>
+            <Grid item xs={12} md={4}>
+              <Typography variant="h3" component="h2">
+                {t('basic_data.institutions.sector')}
+              </Typography>
+              <Typography>
+                {customer?.sector ? t(`basic_data.institutions.sector_values.${customer.sector}`) : '-'}
+              </Typography>
+            </Grid>
+
+            <Grid item xs={12} md={4}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.unique_feide_id')}
+              </Typography>
+              <Typography>{customer?.feideOrganizationDomain ?? '-'}</Typography>
+            </Grid>
+
+            <Grid item xs={12} md={4}>
+              <Typography variant="h3" component="h2">
+                {t('basic_data.institutions.ror')}
+              </Typography>
+              <Typography>
+                {customer?.rorId ? (
+                  <Link href={customer.rorId} target="_blank" rel="noopener noreferrer">
+                    {customer.rorId}
+                  </Link>
+                ) : (
+                  '-'
+                )}
+              </Typography>
+            </Grid>
+          </Grid>
+
+          <Grid container item xs={12} md={12} spacing={2}>
```

Comment:
Mulig hele nestinga kan bort her også. Men 12 på begge xs og md har ingen verdi da de er default.

```suggestion
          <Grid container item spacing={2}>
```